package.json

```diff
@@ -31,13 +35,15 @@
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
+    "@ember/render-modifiers": "^2.0.0",
```

Comment:
should be a `dependency` as you use it in your components

Reply:
https://github.com/concordnow/ember-sidenotes/pull/1/commits/ce222b472e70e06cb7b6d8506f4892a3dfbbb312 :+1: